cypress/fixtures/users/changePasswordUser.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I removed the admin login tests for now because they are the outlier in the tests in regards to resetting data. We can revisit that later. For now, I've replaced them with a normal user login that also tests changing their password. I added a few tests so that I could test the data cleaning that the cypress-rails gem provides.